api/src/canny/canny.handlers.ts

```diff
@@ -59,13 +59,11 @@ export const cannyIdentify = async (
     const user = data[0]!;
 
     winston.info(`Updating profile for ${netId}`);
-    await prisma.studentBluebookSettings.update({
-      where: {
-        netId,
-      },
-      data: {
+    await db
+      .update(studentBluebookSettings)
+      .set({
         // Enable evaluations if user has a school code
-        evaluationsEnabled: Boolean(user.school_code),
+        evaluationsEnabled: user.school_code ? 1 : 0,
```

Comment:
Ugh, does Drizzle not support booleans?

Reply:
I'll take a look into boolean definitions in the schema